js/renovation/ui/scheduler/workspaces/base/date_table/all_day_panel/table_body.tsx

```diff
@@ -0,0 +1,30 @@
+import {
+  Component, ComponentBindings, JSXComponent, OneWay,
+} from 'devextreme-generator/component_declaration/common';
+import { AllDayPanelRow as Row } from './row';
+import { AllDayPanelCell as Cell } from './cell';
+import { getKeyByDateAndGroup } from '../../../utils';
+import { ViewCellData } from '../../../types.d';
+
+export const viewFunction = (viewModel: AllDayPanelTableBody) => (
+  <Row
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    {...viewModel.restAttributes}
+  >
+    {viewModel.props.viewData!.map((_, cellIndex) => (
+      <Cell key={getKeyByDateAndGroup(viewModel.props.viewData![cellIndex].startDate)} />
```

Comment:
For me, it is clearer to do the following:
```
{viewModel.props.viewData!.map((cell) => (
      <Cell key={getKeyByDateAndGroup(cell.startDate)} />
```
We also must take into account `groups` property when generating a key